### PR TITLE
better than stuntguy

### PR DESCRIPTION
**Pull request by JOPHESTUS**

### CODE REVIEW

File: src/main/java/com/dsh105/nexus/command/module/bukkit/TenJavaCommand.java [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
 import java.util.Date;
 
-@Command(command = "tenjava", aliases = {"tj"}, needsChannel = false, help = "Get tenjava points",
-        extendedHelp = {"{b}{p}{c}{/b} - View tenjava donated points"})
+@Command(command = "tenjava", aliases = {"tj", "ten.java"}, needsChannel = false, help = "Get ten.java points",
```
Comment (lol768) [inline src/main/java/com/dsh105/nexus/command/module/bukkit/TenJavaCommand.java:21]: it doesn't give you points. help is misleading

Comment (stuntguy3000) [inline src/main/java/com/dsh105/nexus/command/module/bukkit/TenJavaCommand.java:21]: Improved in #71

**Comment by DSH105** [comment]
Misleading. No pull for you <3

**Comment by JOPHESTUS** [comment]
shutup @lol768

**Comment by lol768** [comment]
u wut m8

**Pull request closed without merging**